app/src/main/res/values/strings.xml

```diff
@@ -932,6 +934,9 @@
     <string name="third_to_last">Third to last chapter</string>
     <string name="fourth_to_last">Fourth to last chapter</string>
     <string name="fifth_to_last">Fifth to last chapter</string>
+    <string name="download_ahead">Download ahead</string>
+    <string name="auto_download_after_reading">Download next unread chapters after reading</string>
+    <string name="download_ahead_info">Download ahead only applies to entries in library and if any chapter read during last session was downloaded</string>
```

Comment:
Less wordy and while simplifies the logic more than what it actually does, it gets the point across
```suggestion
    <string name="download_ahead_info">Download ahead only applies to entries in library and if the last chapter read was downloaded</string>
```